execute/tokendata/usdc/usdc.go

```diff
@@ -4,43 +4,168 @@ import (
 	"context"
 	"fmt"
 
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+	"golang.org/x/exp/maps"
+
 	"github.com/smartcontractkit/chainlink-ccip/execute/exectypes"
+	"github.com/smartcontractkit/chainlink-ccip/internal/reader"
 	"github.com/smartcontractkit/chainlink-ccip/pluginconfig"
 )
 
 type USDCCCTPTokenDataObserver struct {
-	configs []pluginconfig.USDCCCTPObserverConfig
+	configs           pluginconfig.USDCCCTPObserverConfig
```

Comment:
```suggestion
	config           pluginconfig.USDCCCTPObserverConfig
```

Since its just a single config